packages/react/src/components/cookieConsent/CookieConsentContext.tsx

```diff
@@ -0,0 +1,245 @@
+import React, { createContext, useContext, useMemo, useState } from 'react';
+
+import create, { ConsentController, ConsentList, ConsentObject } from './cookieConsentController';
```

Comment:
It might improve readability to rename the create function here to the same name that it has in the file cookieConsentController (createConsentController). It's little easier to find and remember when it's the same imo.